www/api/class/centreon_realtime_hosts.class.php

```diff
@@ -141,20 +141,20 @@ protected function setHostFilters()
         } else {
             $this->viewType = null;
         }
-        if (isset($this->arguments['sortType'])) {
-            if (strtolower($this->arguments['sortType']) === 'asc' ||
-                strtolower($this->arguments['sortType']) === 'desc') {
-                $this->sortType = $this->arguments['sortType'];
+        if (isset($this->arguments['order'])) {
+            if (strtolower($this->arguments['order']) === 'asc' ||
+                strtolower($this->arguments['order']) === 'desc') {
+                $this->order = $this->arguments['order'];
             } else {
-                throw new \RestBadRequestException('Bad sort type parameter');
+                throw new \RestBadRequestException('Bad order parameter');
             }
         } else {
-            $this->sortType = null;
+            $this->order = null;
```

Comment:
```suggestion
            $this->order = 'asc';
```

Reply:
I think, that a default value should be set if none given

Reply:
@sc979  a default value is already given later in the file:
```
            $q = 'ASC';
            if (isset($this->order) && strtoupper($this->order) === 'DESC') {
                $q = 'DESC';
            }
```